components/Card.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+interface Props {
+  background?: 'light' | 'dark'
+  center?: boolean
+  buttonLabel?: string
+  buttonLink?: string
+}
+const props = defineProps<Props>()
+
+const bgColor = computed(() => {
+  switch (props.background) {
+    case 'light':
+      return 'bg-brand-light'
+    case 'dark':
+      return 'bg-brand-blue'
+    default:
+      return 'bg-white'
+  }
+})
```

Comment:
I think it might be a neat idea to make these components dark and light mode aware then use an invert switch to lock the desired mode.

Something to come back to once we've got the first pass laid out 👌

Reply:
I always use dark mode so I am very much in favor of this.